tiago_poses:
- name: 1
  point: [0, 0.085, 0.12, 1.237702952710297e-05, 15.08717871812678, -89.0701061595649]
- name: 2
  point: [0, -0.085, 0.12, 179.9999887144125, 14.05228082906504, 90.92988488492941]